addons/test_mail/tests/test_message_track.py

```diff
@@ -14,8 +14,14 @@ class TestTracking(TestMailCommon):
     def setUp(self):
         super(TestTracking, self).setUp()
 
+        company = self.env['res.company'].create({
```

Comment:
You could use main user company and/or use TestMailMultiCompanyCommon test class if you want to test in mutli company (in which case you could setup MC rules on mail.test.track.currency model).